### PR TITLE
Add odh v1.1.2 OLM bundle

### PR DESCRIPTION
**Pull request by LaVLaS**
Add OLM bundle for the v1.1.2 release of the operator

Signed-off-by: Landon LaSmith <LLaSmith@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    * Seldon v1.2.0 - Open source platform for deploying machine learning models
+    * Argo v2.12.5 - Container-native Workflow Engine
+    * Apache Superset v1.3.2 - **NEW** Open source application for data exploration and visualization
+    * Apache Kafka v2.8.0 - The open source stream processing platform
+    * OpenShift Pipelines v1.3.1 - Cloud-native CI/CD on OpenShift
+    To install one or multiple of these components use the default KfDef provided with the operator.
+
+    ### Kubeflow Components
+    ODH 1.1.2 supports Kubeflow v1.3.0 and some components such as KF Serving and KFP on Tekton from master branch. To install Kubeflow v1.3.0 components please use this example [KfDef](https://raw.githubusercontent.com/opendatahub-io/manifests/v1.3-openshift/distributions/kfdef/kfctl_openshift_v1.3.0_odh.yaml).
+
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:251]: We need to change this to say Kubeflow 1.4 and point to the new 1.4 example kfdef

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+    NOTE: The Open Data Hub distribution of the Kubeflow 1.3 stack  currently runs on OpenShift versions of 4.8 (or lower). For latest updates, please visit [link](https://github.com/opendatahub-io)
+
+    ### OpenShift 4.9
+
+    For installing Open Data Hub on OpenShift 4.9, please use the provided sample [KfDef](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift4.9_odh.yaml) that only includes JupyterHub. We are currently updating all other components to work on OpenShift 4.9 for the latest update please visit [ODH-522](https://issues.redhat.com/browse/ODH-522)
+
+    ### Available Channels
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:258]: Can we specifically state Kubeflow stack does not work on OCP 4.9?

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml (hunk 1 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:251]: Yes. Which example kfdef?

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:260]: I added a sentence about KF on OCP 4.9

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml (hunk 1 above)
Comment (nakfour) [reply deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:251]: This one https://github.com/opendatahub-io/manifests/blob/v1.4.0-rc.2-openshift/openshift/kfdef/kfctl_openshift_v1.4.0_odh.yaml

Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:251]: Updated

File: deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+                },
+                "name": "odh-dashboard"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v1.3-openshift"
+              },
+              {
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:196]: should kf-manifest point to 1.4 branch now?

Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/1.1.2/manifests/opendatahub-operator.v1.1.2.clusterserviceversion.yaml:196]: Updated

**Pull request merged**